src/item.cpp

```diff
@@ -2296,12 +2337,22 @@ int item::get_thickness() const
 
 int item::get_warmth() const
 {
+    int warmed = 1;
     const auto t = find_armor_data();
-    if( t == nullptr ) {
+    if( t == nullptr )
```

Comment:
DDA code needs braced, thanks.  We aim to be earthquake-proof.  (And, more seriously, bracing helps ensure specificity and readability.)